packages/dashboard-frontend/src/store/FactoryResolver/index.ts

```diff
@@ -130,7 +138,12 @@ export const actionCreators: ActionCreators = {
       if (cheEditor) {
         optionalFilesContent['.che/che-editor.yaml'] = cheEditor;
       }
-      const devfile = getDevfile(data, location);
+      let devfile = getDevfile(data, location);
+
+      if (isDevfileV2(devfile) && selectCheDevworkspaceEnabled(state) === false) {
+        devfile = devfileConverter.devfileV2toDevfileV1(devfile);
```

Comment:
we would need to override the message to make it clear that devfile v2 is converted to v1
![Screenshot_20211021_144450](https://user-images.githubusercontent.com/5887312/138270580-c1d23145-85c9-4074-80f4-d75c37a0db0b.png)

Like `devfile 2.x version is found. Converted it to version 1`